doc/howtos/authorized-keys.rst

```diff
@@ -0,0 +1,75 @@
+.. _authorized-keys:
+
+Adding Authorized Keys for Remote ssh Access
+############################################
+
+In Ostro |trade| OS **development** images only, root automatically gets logged in on a local 
+console or serial port connection. By default though, Ostro OS prevents remote logins to your 
+target device.  You can enable remote access as root via ``ssh`` in your development image by installing 
+your personal public key in the ``~root/.ssh/authorized_keys`` file.  This tech note explains how.
+
+You’ll first need to have a private/public key pair on your workstation to enable ``ssh`` access to the target device.  You can use
+an existing host ssh key pair, found in ``$HOME/.ssh``, or you can generate a new one as described in this `generating an ssh key help`_.
+A private/public key pair will be stored in two files for example, ``.ssh/id_rsa`` and ``.ssh/id_rsa.pub``.
+
+.. _`generating an ssh key help`: https://help.github.com/articles/generating-an-ssh-key/
+
```

Comment:
Unfortunately it describes only case for openssh. Many windows users are using putty, so it would be good to get some small description about this case as well.